### PR TITLE
Update messaging around API_DISABLED status

### PR DESCRIPTION
**Pull request by lfkellogg**
Based on feedback from vlukanovich@

### CODE REVIEW

File: firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ class ErrorMessages {
       "The APK failed to install or installation was canceled by the tester.";
 
   static final String API_DISABLED =
-      "The App Distribution Tester API is disabled. It must be enabled in the Google Cloud Console. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.";
+      "The App Distribution Tester API is disabled; you must enable it in the Google Cloud console. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.";
```
Comment (kaibolay) [inline firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java:56]: Could we directly link to the right place in pantheon here? I've seen that in other error messages.

File: firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionException.java [hunk 2]
```diff
@@ -83,11 +83,11 @@ public enum Status {
     /**
      * The Firebase App Distribution Tester API is disabled for this project.
      *
-     * <p>The developer of this app must enable the API in the Google Cloud Console before using the
-     * App Distribution SDK. See the <a
-     * href="https://firebase.google.com/docs/app-distribution/set-up-alerts?platform=android">documentation</a>
-     * for more information. If you enabled this API recently, wait a few minutes for the action to
-     * propagate to our systems and retry.
+     * <p>Before you use the App Distribution SDK in your app, you must enable the API in the Google
```
Comment (kaibolay) [inline firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionException.java:86]: Could we directly link to the right place in pantheon here? I've seen that in other error messages.

**Review by kaibolay — APPROVED** [review]

**Review by jladieu — APPROVED** [review]

File: firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java (hunk 1 above)
Comment (lfkellogg) [reply firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java:56]: Actually we do! We pull it out of the response and append it to the message: https://github.com/firebase/firebase-android-sdk/blob/master/firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java#L261-L263

You can see an example here: [go/fad-iaf-api-disabled](http://go/fad-iaf-api-disabled)

Comment (kaibolay) [reply firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ErrorMessages.java:56]: Thanks!

**Pull request merged**